dev-python/jedi-language-server/jedi-language-server-0.30.2.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+
+DISTUTILS_USE_SETUPTOOLS=pyproject.toml
```

Comment:
Ugh, I just remembered that this is broken.

* The PyPI tarball doesn't include tests but builds/installs successfully with `DISTUTILS_USE_SETUPTOOLS=rdepend`.
* The GitHub tarball includes tests but doesn't include `setup.py` (so `DISTUTILS_USE_SETUPTOOLS=rdepend` doesn't work) but `DISTUTILS_USE_SETUPTOOLS=pyproject.toml` seems to fail to install most of the package. Maybe a bug in `dev-python/pyproject2setuppy` @mgorny?

Reply:
This is now fixed per https://github.com/mgorny/pyproject2setuppy/issues/14